src/library_glemu.js

```diff
@@ -1090,7 +1090,7 @@ var LibraryGLEmulation = {
             func = "textureCube";
             break;
           default:
-            return abort_sanity("Unknown texType: 0x" + texType.toString(16));
+            return abort_sanity("Unknown texType: " + ptrToString(texType));
```

Comment:
Which link flag ensure we include this function? This isn't handled by any of the ones mentioned so far I think.

Reply:
Add settings.LEGACY_GL_EMULATION